src/SubHask/Algebra.hs

```diff
@@ -940,7 +940,7 @@ instance Complemented b => Complemented (a -> b) where
 --
 -- See <https://en.wikipedia.org/wiki/Heyting_algebra wikipedia> for more details.
 --
--- Note that while Heyting algebras are abelian semigroups with respect to ⋀, they are not necessarily cancellative.
+-- Note that while Heyting algebras are abelian semigroups with respect to ⋀, they are not in general cancellative.
```

Comment:
Can you change `⋀` to `(&&)` so that it uses consistent notation with the rest of the module?